test/controllers/application_helper_test.rb

```diff
@@ -21,4 +21,39 @@ def test_add_args_to_url
     assert_equal("/blah\x80",
                  add_args_to_url("/blah\x80", x: "foo\xA0"))
   end
+
+  def test_textile_markup_should_be_escaped
+  	textile = "**Bold**"
+  	escaped = "&lt;div class=&quot;textile&quot;&gt;&lt;p&gt;&lt;b&gt;Bold&lt;/b&gt;&lt;/p&gt;&lt;/div&gt;"
+		assert_equal escaped, escape_html(textile.tpl)
+  end
+
+  def test_conditional_content_tags
+    assert_nil(content_tag_if(nil, :p, "Hello world!"))
```

Comment:
Just to be clear, the 1 assertion/test is a good rule, but it doesn't always make complete sense.  The real rule is a given test should test one thing.  If you have more than one assertion in a test it's worth making sure it doesn't violate the real rule.  Examples where more than one assertion is fine are cases where more than one thing happen as the result of a single action.  For example you may want to test that an action created a particular objects and the it redirected to a particular page.  Another time may use additional assertions in a single test is if I want to make sure that some precondition is setup before the action of the test happens.